translations/en.json

```diff
@@ -1223,8 +1225,9 @@
     "save-password-unavailable": "Set device passcode to save password",
     "save-password-unavailable-android": "Save password is unavailable: your device may be rooted or lacks necessary security features.",
     "scan-or-enter-sync-code": "Scan or enter sync code",
-    "scan-qr": "Scan QR code",
-    "scan-qr-code": "Scan a QR code with a wallet address",
+    "scan-qr": "Scan QR",
```

Comment:
i meant we need only one version, no need to have both

Reply:
it will update the text on wallet screen. Will that be okay?

Reply:
yes, thanks

Reply:
great, Thanks for review